src/sentry/search/snuba/executors.py

```diff
@@ -787,15 +782,14 @@ def query(
         # are no other Snuba-based search predicates, we can simply
         # return the results from Postgres.
         if (
-            allow_postgres_only_search
-            and cursor is None
+            cursor is None
             and sort_by == "date"
             and
             # This handles tags and date parameters for search filters.
             not [
                 sf
                 for sf in (search_filters or ())
-                if sf.key.name not in self.postgres_only_fields.union(["date", "timestamp"])
```

Comment:
Dont use postgres-only search when the query explictly is filtering on event timestamp. In this case, we have to go through snuba to find the events within the timestamp range.